shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -382,7 +395,11 @@ private void hideTextInput(View view) {
   void setTextInputClient(int client, TextInputChannel.Configuration configuration) {
     // Call notifyViewExited on the previous field.
     notifyViewExited();
-    inputTarget = new InputTarget(InputTarget.Type.FRAMEWORK_CLIENT, client);
+    if (canShowTextInput()) {
```

Comment:
`canShowTextInput` depends on `this.configuration` which hasn't been updated at this point.

Reply:
The result of blindly fixing failing tests :facepalm: